packages/app/setup.py

```diff
@@ -63,6 +63,22 @@ def get_var(var_name):
     'galaxy.tools.toolbox.lineages',
     'galaxy.tools.util',
     'galaxy.tools.util.galaxyops',
+    'galaxy.tool_shed',
+    'galaxy.tool_shed.galaxy_install',
+    'galaxy.tool_shed.galaxy_install.datatypes',
+    # TODO:
+    # 'galaxy.tool_shed.galaxy_install.grids',
```

Comment:
This can be removed from Galaxy after the separation, right ?

Reply:
These are still used by the web controllers in this branch but are deleted from #8830. So I think when we clean out the web controllers they can just disappear and I didn't bother to migrate them. I don't understand this to be tied to the tool shed separation since the tool shed doesn't depend on them - it is more about the tests being replaced or removed.

Reply:
OK, I assumed that the toolshed uses them (because I also thought Galaxy doesn't use those controllers anymore). Fine to delete them in #8830.